### ISSUE TITLE
how to draw right rect on axialSlice view

### ISSUE THREAD

**Issue by hu3402379** [rii-mango/Papaya#125]
@martinezmj 
Hi, I have a question and hope to help me。
Now, i have some coordinates like [{x: 10, y: 50}, {x: 50, y: 50}, {x: 50, y: 100}, {x:10,y: 50}].
and these coordinates are relative to the source dicom's position.

when I set params['radiological'] = true. 
and i add a prototype function like:

> papaya.viewer.Viewer.prototype.translateCoordinate = function (coordinate, type) {
    if (type == 'x') {
        return (this.axialSlice.finalTransform[0][2] + (coordinate + 0.5) * this.axialSlice.finalTransform[0][0])
    } else {
        return (this.axialSlice.finalTransform[1][2] + (coordinate + 0.5) * this.axialSlice.finalTransform[1][1])
    }
}

this is draw rect part code:
> context.lineWidth = 1
                            context.moveTo(that.translateCoordinate(rect[0].x, 'x'), that.translateCoordinate(rect[0].y, 'y'))
                            arr.forEach(function (item) {
                                context.lineTo(that.translateCoordinate(item[0], 'x'), that.translateCoordinate(item[1], 'y'))
                            })
                            context.closePath()
                            context.stroke()

**and the rectangle position is reversed in the image, like:**
![image](https://user-images.githubusercontent.com/7867792/41962441-a632b108-7a27-11e8-8ec4-ab85604a062e.png)

**and right position should be like:**

![image](https://user-images.githubusercontent.com/7867792/41962555-ec1b5fbc-7a27-11e8-9128-0d62983d58e3.png)

I do not know how to solve this problem, please help me, thank you~~

**Comment by hu3402379** [comment]
Hi @martinezmj, 
Did you notice this issue? 
Can you help me or give me some ideas?

thanks.

**Comment by JerryGit1** [comment]
Hello:

I have encountered a problem similar to yours. Have you solved it? I want to know the relationship between the coordinates of XYZ and the offset in canvas.

At present, the problems I encounter are:

I added a new function based on papaya to draw graphics on the canvas. The coordinates are positioned according to the offset of the system itself. I found that the offset positioning is inconsistent with the XY of papaya. So I would like to ask the relationship between them and how to convert them. Or how to embed my drawing into papaya.
